shared/push/push.js.flow

```diff
@@ -0,0 +1,17 @@
+// @flow
+import {Component} from 'react'
+
+import type {TokenType} from '../constants/push'
+
+export type Props = {
+  prompt: boolean,
```

Comment:
Is this only used to determine to show the push prompt?

This should be the decision of the parent, I don't think it's very common to have the child decide not to render itself depending on a prop